src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -64,5 +71,49 @@ private static Process GetProcessById(int processId)
                 return null;
             }
         }
+
+        private static string GetArgs(Process process)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                try
+                {
+                    using (ManagementObjectSearcher searcher = new ManagementObjectSearcher($"SELECT CommandLine from Win32_Process WHERE ProcessId = {process.Id}"))
+                    {
+                        using (ManagementObjectCollection objectCollection = searcher.Get())
+                        {
+                            return objectCollection.Cast<ManagementBaseObject>().SingleOrDefault()?["CommandLine"]?.ToString().Split("  ")?.Last().Replace("\"", "");
+                        }
+                    }
+                }
+                catch (Exception ex) when (ex is Win32Exception || ex is InvalidOperationException)
+                {
+                    return "[Elevated process - cannot determine command line arguments]";
+                }
+
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                try
+                {
+                    string commandLine = File.ReadAllText($"/proc/{process.Id}/cmdline");
+                    if(!String.IsNullOrWhiteSpace(commandLine))
+                    {
+                        //The command line may be modified and the first part of the command line may not be /path/to/exe. If that is the case, return the command line as is.Else remove the path to module as we are already displaying that.
+                        if(commandLine.Split('\0').FirstOrDefault() == process.MainModule.FileName)
+                        {
+                            return String.Join("", commandLine.Split('\0').Skip(1)).Replace("\0", "");
```

Comment:
I think the `Replace` here is redundant since the `Split` will return an array of strings without `\0`.